documentation/modules/con-kafka-connect-authentication.adoc

```diff
@@ -42,17 +45,16 @@ spec:
   # ...
 ----
 
-== SASL based SCRAM-SHA-512 authentication
+To use this method, see xref:proc-configuring-kafka-connect-authentication-tls-{context}[].
 
-To configure Kafka Connect to use SASL-based SCRAM-SHA-512 authentication, set the `type` property to `scram-sha-512`.
-This authentication mechanism requires a username and password.
+== SASL-based SCRAM-SHA-512 authentication
 
-* Specify the username in the `username` property.
-* In the `passwordSecret` property, specify a link to a `Secret` containing the password. The `secretName` property contains the name of the `Secret` and the `password` property contains the name of the key under which the password is stored inside the `Secret`.
+To configure Kafka Connect to use SASL-based SCRAM-SHA-512 authentication, set the `authentication.type` property to `scram-sha-512`.
+This authentication mechanism requires a client username and password, which are contained in a Secret.
```

Comment:
It is only the password what is in secret. The username is directly in the CR (see the example below.).